test/integration/change-streams/change_stream.test.ts

```diff
@@ -1262,7 +1262,9 @@ describe('Change Streams', function () {
           }
           req.reply({ ok: 1 });
         });
-        const client = this.configuration.newClient(`mongodb://${mockServer.uri()}/`);
+        const client = this.configuration.newClient(`mongodb://${mockServer.uri()}/`, {
+          serverApi: null // TODO(NODE-3807): remove resetting serverApi when the usage of mongodb mock server is removed
```

Comment:
From the description:

> - Change some clients to mocked servers configured with an API version to not set a serverApi, because [mocked servers do not support OP_MSG](https://github.com/mongodb/node-mongodb-native/blob/main/test/tools/mongodb-mock/src/server.js#L310) and fail when run on latest_server_v1_api. There is the existing ticket to remove usage of mongodb mock server ([NODE-3807](https://jira.mongodb.org/browse/NODE-3807)).

Does that clear this up?